src/push/monitor.ts

```diff
@@ -184,7 +185,15 @@ export async function createLightweightMonitors(
   let warnOnce = false;
   const monitors: Monitor[] = [];
   for (const file of lwFiles.values()) {
-    const content = await readFile(file, 'utf-8');
+    // First check encoding and warn if any files are not the correct encoding.
+    const bufferContent = await readFile(file);
+    const isUtf8 = utf8Validate(bufferContent);
```

Comment:
is this necessary?, I believe you can add a try catch on `readFile` and log the failures?

Reply:
@vigneshshanmugam in my case with a 'utf-16le' `readFile(file, 'utf-8')`  read the file without errors, however the resulting string was jumbled.  This caused the yaml to not parse and the file to be skipped.

Reply:
Gotcha, can you remove the dep and use the built in Buffer.isUtf8 https://nodejs.org/api/buffer.html#bufferisutf8input

Reply:
@vigneshshanmugam I updated the code to use the standard library function.  I also updated the `@types@node` dev dependencies.  The types update was needed to get the definition for the utf8 function, however it was also needed simply to be in lockstep with the engine requirement.